chinese/wg-diversity-inclusion/focus-areas/project-and-community/psychological-safety.md

```diff
@@ -1,112 +1,124 @@
 # 心理安全
 
-问题: 社区成员在社区中感到安全的程度，包括参与提交贡献，影响变化，带来他们真实的自我，以及一般性的参与项目？
+问题：社区成员在社区中感到安全的程度，包括参与提交贡献，影响变化，带来他们真实的自我，以及一般性地参与项目？
```

Comment:
Since you already updated the description of question, would you please update this question to readme
https://github.com/chaoss/translations/blob/main/chinese/wg-diversity-inclusion/focus-areas/project-and-community/README.md ?

Reply:
Hi, @eyehwan .Thanks for your suggestion. I have finished it.